src/mono/sample/wasm/blazor-frame/blazor.csproj

```diff
@@ -10,8 +10,9 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.AspNetCore.Components.WebAssembly" Version="8.0.0" />
-    <PackageReference Include="Microsoft.AspNetCore.Components.WebAssembly.DevServer" Version="8.0.0" PrivateAssets="all" />
+    <!-- TODO un-pin this when it's possible -->
```

Comment:
is there better way than specific alpha version @akoeplinger ?
something that would flow the latest Blazor

Reply:
I think we will need to start creating the template on the fly with the `dotnet new blazorwasm` command and patching it with required changes.

The browser template measurement would benefit from the same approach too.

Reply:
> something that would flow the latest Blazor

We would need a darc dependency, wouldn't we?

> The browser template measurement would benefit from the same approach too.

Why would would it be benefitial for wasmbrowser?

Reply:
> We would need a darc dependency, wouldn't we?

Yeah but it would create a circular dependency between aspnetcore <-> runtime which we usually avoid.

Another way that should work is looking at the `LatestRuntimeFrameworkVersion` key on the `Microsoft.AspNetCore.App` `KnownFrameworkReference` which matches the current TFM.

Reply:
> @radekdoulik Why would would it be benefitial for wasmbrowser?

Would it even help keep us up to date with latest or it's the same darc flow problem?